board/main.c

```diff
@@ -552,6 +553,11 @@ int main() {
 
   __enable_irq();
 
+  // if the error interrupt is enabled to quickly when the CAN bus is active
+  // something bad happens and you can't connect to the device over USB
+  delay(10000000);
+  CAN1->IER |= CAN_IER_ERRIE | CAN_IER_LECIE;
```

Comment:
Without this delay, if you hook a panda into an active can bus (which causes `can_init` to loop on setting the speed as opposed to the error interrupt triggering to iterate on setting the speed) you can't connect to it over USB (even though the lights are all blinking as if it detected the speed and is receiving messages)